### ISSUE TITLE
Test compiling zstddec or xzdec (wasm and asm versions) with MINIMAL_RUNTIME

### ISSUE THREAD

**Issue by Jaifroid** [kiwix/kiwix-js#745]
This would be to eliminate the warning shown in console about a streaming WASM compilation error. Although the current implementaiton provides a fallback to ArrayBuffer instnatiation, it would be good to avoid the warning if we can, by setting the appropriate Emscripten flags at compile time.

For further details, see discussion here:

https://github.com/kiwix/kiwix-js/pull/740#issuecomment-885163078

and here:

https://github.com/kiwix/kiwix-js/pull/740#issuecomment-886094851

**Comment by kelson42** [comment]
@mossroy @Jaifroid Does this ticket still makes sense considering that we are following the emscripten road?

**Comment by Jaifroid** [comment]
@kelson42 I would say this should be kept until we are sure that we can use the libzim version in most modern contexts. There are still some issues to resolve with that, albeit @mossroy has resolved the major blocker.

**Comment by kelson42** [comment]
@Jaifroid I guess we can now close this then?

**Comment by Jaifroid** [comment]
Yes, I don't think realistically we're going to be recompiling XZ or ZSTD given that the versions we have work fine with old browsers and don't cause any issues. We can close now.
